src/control/server/config.go

```diff
@@ -53,6 +53,14 @@ const (
 type networkProviderValidation func(string, string) error
 type networkNUMAValidation func(string, uint) error
 
+// AgentCfg elements are used by the libdaos clients to help initialize CaRT.
+// These settings bring coherence between the client and server network configuration.
+type AgentCfg struct {
+	Provider        string
+	CrtCtxShareAddr uint32
+	CrtTimeout      uint32
+}
```

Comment:
As I was looking at the rest of the code, it occurred to me that maybe AgentCfg isn't the right name for this. SharedNetworkCfg perhaps? Simply NetworkCfg? Since it appears to be CaRT specific, maybe CartCfg?

Reply:
Changed to ClientNetworkCfg